### DISCUSSION TITLE
{{file_extension}} variable - needed or not?

### DISCUSSION THREAD

**Discussion by Taitava** [Taitava/obsidian-shellcommands#121] [Ideas - done & released]
Currently there are:
 - `{{title}}` to get file name without the extension.
 - `{{file_name}}` to get file name with the extension.

Would there be use for `{{file_extension}}` that would return usually `.md`? Obsidian can also show image files and PDFs, so I think in some cases it might be beneficial to be able to check what the extension is - although it can be determined from `{{file_name}}` using regex.

One detail: ~~the value would include a preceding dot `.`, but could be opted-out with an argument like `:no-dot`.~~
Edit:
 - `{{file_extension:no-dot}}`: No preceding dot is included.
 - `{{file_extension:with-dot}}`: A preceding dot is included if the extension is not empty.

**Comment by FelipeRearden** [comment]
I like the idea @Taitava !!!!
